docs/source/how-to/understand-keys-and-how-to-change-them.md

```diff
@@ -0,0 +1,146 @@
+---
+jupytext:
+  text_representation:
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
+# Understanding intake-ESM keys and how to use them
+
+Intake-ESM helps with aggregating your datasets using some `keys`. Here, we dig into what exactly these keys are, how they are constructed, and how you can change them. Understanding how this work will help you control how your datasets are merged together, and remove the mystery behind these strings of text.
+
+## Import packages and spin up a Dask cluster
+
+We start first with importing `intake` and a `LocalCluster` from `dask.distributed`
+
+```{code-cell} ipython3
+import intake
+from distributed import LocalCluster
+
+cluster = LocalCluster()
```

Comment:
```suggestion
from distributed import Client

cluster = Client()
```

Reply:
Yeah - good point. This is a better approach